npm/private/npm_translate_lock_state.bzl

```diff
@@ -93,6 +103,12 @@ def _init_common_labels(rctx, label_store):
     if attr.pnpm_lock:
         label_store.add("pnpm_lock", attr.pnpm_lock, seed_root = True)
         label_store.add("lock", attr.pnpm_lock)
+
+        # root package.json file
+        root_package_json_label = str(attr.pnpm_lock).replace(PNPM_LOCK_FILENAME, PACKAGE_JSON_FILENAME)
+        if root_package_json_label in rctx.attr.data:
```

Comment:
hmmm. it is possible that this won't work if someone doesn't get their labels straight:
```
pnpm_lock = "//path/to:pnpm-lock.yaml",
data = ["//:path/to/package.json"],
```

Reply:
Good catch. Fixed.